### PR TITLE
update doc for GCRCatalogs 1.3.3; add "mirror" section to how_to_ackn…

### PR DESCRIPTION
**Pull request by JoanneBogart**
…owledge

Additional edits  coming soon for the new image data.  These changes probably will involve at least
getting_started.md
download.md
dc2_sim_sky_survey.md

### CODE REVIEW

File: web/portal/templates/doc/how_to_acknowledge.md [hunk 1]
```diff
@@ -21,3 +21,11 @@ If you use the `GCRCatalogs` package in a publication,
 we ask that you cite the following publication.
 
 * [DESCQA Paper](https://ui.adsabs.harvard.edu/abs/2018ApJS..234...36M/abstract)
+
+### Mirrored Data
```
Comment (yymao) [inline web/portal/templates/doc/how_to_acknowledge.md:25]: How about "Mirroring data for community/non-personal use"?  Just want to make the title a bit more descriptive.

File: web/portal/templates/doc/install_gcr.md [hunk 2]
```diff
@@ -18,6 +18,10 @@ However, please consider switching to the conda-forge channel;
 see [instructions](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge)
 and [why](https://conda-forge.org/docs/user/tipsandtricks.html#using-multiple-channels).
 
+*NOTE:* In order to access the latest catalogs (as of June, 2021) you need GCRCatalogs
```
Comment (yymao) [inline web/portal/templates/doc/install_gcr.md:21]:
```suggestion
*NOTE:* In order to access the latest catalogs (as of June 2021) you need `GCRCatalogs`
```

File: web/portal/templates/doc/install_gcr.md [hunk 3]
```diff
@@ -18,6 +18,10 @@ However, please consider switching to the conda-forge channel;
...
+version v1.3.3 or later. When using either of the install methods described below you will get a suitable released version. If you have already installed GCRCatalogs for an earlier
```
Comment (yymao) [inline web/portal/templates/doc/install_gcr.md:22]:
```suggestion
version v1.3.3 or later. When using either of the install methods described below you will get a suitable released version. If you have already installed `GCRCatalogs` for an earlier
```

**Review by yymao — COMMENTED** [review]
Thanks @JoanneBogart. Looks good. I added a few minor early review comments.

File: web/portal/templates/doc/how_to_acknowledge.md (hunk 1 above)
Comment (JoanneBogart) [reply web/portal/templates/doc/how_to_acknowledge.md:25]: Yes, this should be more specific.

**Comment by JoanneBogart** [comment]
I've added text concerning coadd images and deprecation of v1 plus other small changes.

File: web/portal/templates/doc/dc2_sim_sky_survey.md [hunk 4]
```diff
@@ -10,10 +10,14 @@ To learn more about the DC2 Simulated Sky Survey, you can read:
 * [DESC DC2 Data Release Note](https://arxiv.org/abs/2101.04855):
   a brief note about the released dataset, including data file format, data partition scheme, and schema tables.
 
-Follow the steps below to access DC2 Simulated Sky Survey.
+Two kinds of datasets are provided: catalogs and images. Follow the instructions below for the type you wish to download.
```
Comment (yymao) [inline web/portal/templates/doc/dc2_sim_sky_survey.md:13]:
```suggestion
Two kinds of datasets are provided: [catalogs](#catalogs) and [images](#images). Follow the instructions below for the type you wish to download.
```

File: web/portal/templates/doc/dc2_sim_sky_survey.md [hunk 5]
```diff
@@ -43,8 +47,15 @@ print(GCRCatalogs.load_catalog('desc_dc2_run2.2i_dr6_object').available_tracts)
...
 You can find examples of using `GCRCatalogs` in our [tutorial notebooks](https://github.com/LSSTDESC/desc-data-portal/tree/main/notebooks).
 
 If you're planning to run the example notebooks and don't already have JupyterLab on your laptop, see [these instructions](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html).
+
+## Images
+
+Unlike the catalog datasets, it is not possible to choose a subset of the image collection datasets.
+Two image collections are provided. For a detailed description of the kinds of image files included, please see Appendix C of the [DESC DC2 Data Release Note](https://arxiv.org/abs/2101.04855).
+
+`DC2 coadd dr6 v2` includes all of these images belonging to tracts 3828 and 3829. `DC2 coadd dr6 v2 small` includes images for a single patch for each of these tracts.
```
Comment (yymao) [inline web/portal/templates/doc/dc2_sim_sky_survey.md:61]:
```suggestion
Two image collections (of different sky coverages) are provided. `DC2 coadd dr6 v2` includes all coadded images belonging to tracts 3828 and 3829. `DC2 coadd dr6 v2 small` includes coadded images for a single patch for each of these tracts.
```

File: web/portal/templates/doc/dc2_sim_sky_survey.md [hunk 6]
```diff
@@ -43,8 +47,15 @@ print(GCRCatalogs.load_catalog('desc_dc2_run2.2i_dr6_object').available_tracts)
...
+#### Step 4: Follow the notebooks to access data
 
 You can find examples of using `GCRCatalogs` in our [tutorial notebooks](https://github.com/LSSTDESC/desc-data-portal/tree/main/notebooks).
 
 If you're planning to run the example notebooks and don't already have JupyterLab on your laptop, see [these instructions](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html).
+
+## Images
+
+Unlike the catalog datasets, it is not possible to choose a subset of the image collection datasets.
+Two image collections are provided. For a detailed description of the kinds of image files included, please see Appendix C of the [DESC DC2 Data Release Note](https://arxiv.org/abs/2101.04855).
```
Comment (yymao) [inline web/portal/templates/doc/dc2_sim_sky_survey.md:59]:
```suggestion
For a detailed description of the coadded image and associated files included, please see Appendix C of the [DESC DC2 Data Release Note](https://arxiv.org/abs/2101.04855).
```

File: web/portal/templates/transfer.jinja2 [hunk 7]
```diff
@@ -14,12 +14,15 @@
...
       <a href="{{url_for('profile')}}" title="Click to change source endpoint on the Profile page.">{{- 'ANL' if session.source_endpoint == 0 else 'NERSC' -}}</a>.
-      <strong>Or</strong>, click on a dataset name to browse and choose individual files for transfer.
+      <strong>Or</strong>, for all but the coadd datasets, click on a dataset name to browse and choose individual files for transfer.
       See <a href="{{url_for('render_doc', doc_name='download')}}">here</a> for detailed instructions.
     </p>
-
     <p>
-      To learn more about a dateset, click the "doc" link next to the dataset name.
+      The <b>v1</b> datasets are deprecated. Under normal circumstances you
+      should choose the corresponding <b>v2</b> dataset instead.
```
Comment (yymao) [inline web/portal/templates/transfer.jinja2:22]: I'll probably implement some javascript tricks to hide v1 by default. I'll leave this here and edit them accordingly when I implement the trick.

**Review by yymao — CHANGES_REQUESTED** [review]
Thanks @JoanneBogart -- I made some suggested changes to further clarify the text.

File: web/portal/templates/transfer.jinja2 (hunk 7 above)
Comment (JoanneBogart) [reply web/portal/templates/transfer.jinja2:22]: Ok.

**Review by yymao — APPROVED** [review]
LGTM now. Thanks @JoanneBogart

**Pull request merged**